.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     rev: 19.3b0
     hooks:
       - id: black
--   repo: https://gitlab.com/pycqa/flake8
+-   repo: https://gitlab.com/pycqa/flake8.git
```

Comment:
Was this intended? Is this needed?

Reply:
This is required to pull flake8 on CentOS 7 as documented in https://github.com/pre-commit/pre-commit/issues/1206

Reply:
Nope, just wondering why that file was changed. Getting that bug linked is good documentation for us if something goes awry.

Thanks for your contribution to the package!